lib/yardstick/v2_client/grant.rb

```diff
@@ -0,0 +1,40 @@
+require 'yardstick/v2_client/remote_model'
+require 'yardstick/cacheable'
```

Comment:
Are these seriously not `required` in a more central location???

Reply:
yeah this is the pattern `user.rb`uses, which i modelled this after...i wonder if it's actually necessary lol.